dev/automated_tests/flutter_test/flutter_gold_test.dart

```diff
@@ -34,26 +35,41 @@ void main() {
       environment: <String, String>{'FLUTTER_ROOT': '/flutter'},
       operatingSystem: 'macos'
     ),
+    log: log.add,
   );
 
   test('Local passes non-existent baseline for new test, null expectation', () async {
+    log.clear();
     expect(
       await comparator.compare(
         Uint8List.fromList(_kFailPngBytes),
         Uri.parse('flutter.new_golden_test.1.png'),
       ),
       isTrue,
     );
+    expect(log, <String>[
+      // ignore: no_adjacent_strings_in_list
```

Comment:
Can this just be written to avoid needing an ignore?

Reply:
pulled the string into a constant and made the list just contain the constant.